### ISSUE TITLE
[Bug] NC_ERROR max StreamId reached

### ISSUE THREAD

**Issue by fel88** [openvinotoolkit/openvino#2482]
##### System information (version)

- OpenVINO=> commit 023e7c2c3f8a8ac83564db09799d2049115d9cf6 (HEAD, tag: 2020.4)
- Operating System / Platform => Orange's Ubuntu Bionic Image v1.3
- System info=> Linux orangepi4 4.4.179-g7b2dd521-dirty #5 SMP Tue Mar 24 14:07:36 CST 2020 aarch64 aarch64 aarch64 GNU/Linux
##### Detailed description

Hi, my MYRIAD device (Neural Compute Stick 2) connects successfully only after 5-10 attempts (randomly). In others cases I get the error: 'Max streamId reached'.

I am using single-board computer OrangePi4 with Rockhip RK3399 SoC. I'm running Orange's Ubuntu Bionic Image v1.3 from here (https://drive.google.com/drive/folders/1Ryg8N8O67lqoKWRnXzuYypgof7HtWuAa). Linux kernel is 4.4.179. Intel Neural Compute Stick 2 is inserted in the top USB 3.0 port. I'm not using any USB Hubs. I do not use any virtualization. The architecture of binaries is ARM 64-bit (aarch64).

![bug2](https://user-images.githubusercontent.com/15663687/94782690-46c1fb00-03d4-11eb-9a1e-077e3d716962.png)
![bug](https://user-images.githubusercontent.com/15663687/94782687-4590ce00-03d4-11eb-9f65-ce58b9936d7f.png)

If the device is plugged to the USB 2.0 port, the error doesn't happen

##### Steps to reproduce
Minimal sample to reproduce:
```c++
#include <iostream>
#include <inference_engine.hpp>

using namespace std;

int main(int argc,char**argv)
{
     InferenceEngine::Core ie;
     InferenceEngine::CNNNetwork cnnNetwork;
     cnnNetwork=ie.ReadNetwork(argv[1]);
     InferenceEngine::ExecutableNetwork network;
     network = ie.LoadNetwork(cnnNetwork, "MYRIAD");
     return 0;
}
```
Any model (.xml, .bin) can be used as input for this program, for example: text-recognition-0012 (from OpenVINO model downloader)

How to fix it?

**Comment by dkurt** [comment]
Hi!

Also experienced this issue at https://github.com/dkurt/vineyard/.

Please try the latest master branch of OpenVINO - problem disappeared for me.

**Comment by fel88** [comment]
Thank you, but unfortunately it didn't help.
I tried both (latest master and releases/2021/1). The problem still remains.
Any other ways I can try?

**Comment by fel88** [comment]
Solved by updating Linux Kernel to version 5.8
